functions/package.json

```diff
@@ -10,9 +12,29 @@
     "node": "14"
   },
   "bin": {
-    "import": "./src/bin/import.js"
+    "app-search-firestore-extension": "./lib/bin/import.js"
   },
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
```

Comment:
I had to add this so that all of our built files were actually added to to the bundle in npm.